content/docs/started/k8s/kfctl-k8s-istio.md

```diff
@@ -77,6 +77,8 @@ Notes:
   a local version of the configuration YAML file which you can further
   customize if necessary.
 
+* **Disabling Istio Installation** - In the configuration file {{% config-file-istio-dex %}} you may choose to not install Istio by removing the applications `istio-crds` and `istio-install`, if your Kubernetes cluster has an existing Istio installation.
+
```

Comment:
While you're here, please would you remove the following line from the top of this file: **Maintainer and supporter: Kubeflow community**. That line is no longer relevant, because both configs are supported by the Kubeflow community.

Reply:
Sure